requirements.txt

```diff
@@ -6,4 +6,4 @@ click
 sphinx
 sphinx_rtd_theme
 uvloop
-gunicorn
\ No newline at end of file
+gunicorn==19.9.0
```

Comment:
I assume the update did not work as well, as the issue should have been fixed.

Reply:
Nope, that was just forgotten from the merge – fixed.